metadata_updater/metadata_updater.py

```diff
@@ -142,10 +142,12 @@ def get_metadata(layer, dir, overwrite):
         file_exists(file_destination)
 
     if layer.metadata != None:
-        layer.metadata.get_xml(file_destination)
-        return file_destination
-    else:
-        return None
+        try:
+            layer.metadata.get_xml(file_destination)
+            return file_destination
+        except:
+            print("No Metadata Found for: ", layer)
```

Comment:
what are your thoughts on changing this and L347 to logs? 
For example, logger.info(...)